index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
### Your site has not style    :sunglasses:
---

 - _Without a style.css file your site will not have any style applied._-
---
### Fix

Adding `<link type="style-sheet" src=".style.css"/>` would solve the problem.

Reply:
Ops. Fixed